content/W2016/W16-event-gitworkshop.md

```diff
@@ -0,0 +1,36 @@
+Title: Technical Workshop: Intro to Git and Open-Source
+Date: 2016-03-03 14:00
+Category: Events
+Tags: Blackberry, workshop, git, open-source
```

Comment:
Also the "open source" tag [doesn't have a dash](http://wics.uwaterloo.ca/tags/open-source/)